README.md

```diff
@@ -1,7 +1,26 @@
-# Schulich-Delta-CCS-Test-Tool
-This test MBED code aims to simulate input values for the CCS.
-These values wil be ones the CCS would typically expect from Schulich Delta.
-The test MBED will then return output values or pass/fail based on how the CCS handled the inputs.
+# Embedded Test Tool
+
+This is an embedded system testing tool which is powered by an MBED LPC1768.
+It is built by the University of Calgary Solar Car Team with the purpose of testing the Gen 5 Solar Car embedded systems.
+The embedded test tool will send out signals connected to a different board, and check the signals coming back.
+The test MBED is controlled via serial and will output pass, fail, and information messages based on how the target board handled the inputs.
+
+## Compiling
+
+To compile, you will need the `arm-none-eabi-g++` compiler.
+You can get it from https://github.com/adamgreen/gcc4mbed.
+Run whichever installer script matches your OS, then run the `BuildShell` script.
+This will temporarily add the compiler to your PATH, you can then compile.
+You can use the `makeall.sh` script to compile both the tests and source and then run the tests.
```

Comment:
You should specify that the makeall.sh script is in the Embedded Test Tool files, I spent forever looking through the adamgreenmbed folder haha